### ISSUE TITLE
move from /settings/user to /settings/user/privacy/

### ISSUE THREAD

**Issue by alexanderdd** [nextcloud/data_request#241]
I think users would search for the functionality under the privacy tab, not at the bottom of their personal info.

So I propose to move the data export and deletion request sections from cloud.yourcloud.com/settings/user to .../settings/user/privacy/

**Comment by max-nextcloud** [comment]
@nextcloud/designers what's your take on this?

**Comment by jancborchardt** [comment]
Yeah good point @alexanderdd! Especially considering that we introduced this before we had the Privacy settings and now that we have way more Personal settings, it makes more sense in "Privacy". :)
